README.rst

```diff
@@ -142,10 +81,115 @@ Nested Configuration
++-------------------+---------------+-----------------+-------------------------------------------------+-------------------------------------------------------------------+
+| ``source``        | yes           | yes             | ``os.environ``                                  | Dictionary that the value will be loaded from.                    |
++-------------------+---------------+-----------------+-------------------------------------------------+-------------------------------------------------------------------+
+| ``wanted_type``   | yes           | yes             | ``str``                                         | Desired Python type of the attribute's value.                     |
+|                   |               |                 |                                                 |                                                                   |
+|                   |               |                 |                                                 | On the variable level it is specified via a type annotation on    |
+|                   |               |                 |                                                 | the attribute: ``my_var_1: my_wanted_type``.                      |
+|                   |               |                 |                                                 |                                                                   |
+|                   |               |                 |                                                 | However it can be also specified on the class level, then it acts |
+|                   |               |                 |                                                 | as a default when the annotation is not provided:                 |
+|                   |               |                 |                                                 |                                                                   |
+|                   |               |                 |                                                 | ``class MyConfig(ecological.Config, wanted_type=int, ...)``       |
++-------------------+---------------+-----------------+-------------------------------------------------+-------------------------------------------------------------------+ 
+
+Following rules apply when options are resolved:
+
+- in the case of specyfing options on both levels (variable and class)
+  the variable ones take precedence over class ones,
+- when some options are missing on the variable level their defaults
+  are taken from the class level,
+- it is not necessary to assign an ``ecological.Variable`` instance to
+  change the behavior; it can still be changed on the class level (globally).
+
+Autoloading
+------------
+It is possible to defer/disable autoloading (setting) of variable values by specifying the ``autoload`` option on class definition.
```

Comment:
I would give more context by adding a paragraph before this one. Something like:

> By default, the values of the attributes of your Ecological configuration class are automatically loaded when the class is created.

Reply:
I am not sure about this one. It's already mentioned once in `Getting started` section and that would also repetition here in regard to the next subsection. Also it's in `Advanced` section. WDYT?

Reply:
After a second read, I agree with you — the _Getting started_ section is not far away. I got biased by only reading the diff. :+1: